controller/src/beerocks/master/db/db.cpp

```diff
@@ -1232,6 +1232,48 @@ const std::vector<beerocks_message::sWifiChannel> db::get_hostap_supported_chann
     return n->hostap->supported_channels;
 }
 
+/**
+ * @brief Add supported operating class to the database.
+ * Currently this function is a wrapper which converts the operating
+ * class to a set of supported channels and updates the list of currently
+ * supported channels.
+ * 
+ * @param mac radio mac
+ * @param info operating class info from the tlvApRadioBasicCapabilities TLV
+ * @return true on success
+ * @return false on failure
+ */
+bool db::add_hostap_supported_operating_class(std::string mac, wfa_map::cOperatingClassesInfo &info)
```

Comment:
So, this is *not* what I meant :-/

I would prefer to not expose the details about how things happen to be structured in the MAP TLVs to the database. So I would expand the structure into an operating class and transmit power argument and a vector of statically non-operable channels.

(nitpick)